scarb/tests/local_registry.rs

```diff
@@ -150,6 +150,9 @@ fn publish() {
             .success()
             .stdout_matches(formatdoc! {r#"
             [..] Packaging {name} v{version} ([..])
+            [..] Verifying {name}-{version}.tar.zst
+            [..] Compiling {name} v{version} ([..])
+            [..]  Finished release target(s) in [..]
```

Comment:
Do we need to see this? 🤔

Reply:
That's how Cargo behaves

<img width="555" alt="image" src="https://github.com/software-mansion/scarb/assets/37044072/558cf50c-8cb8-4a01-91d8-16702192f259">

Reply:
Oh, ok, haven't realize!